_includes/manuals/2.1/1.2_release_notes.md

```diff
@@ -389,6 +389,9 @@ By default it is deployed using [systemd socket activation](http://0pointer.de/b
 
 ### Deprecations
 
+* Puppetrun feature is being removed from Foreman core. This is one of many steps to achieve having Puppet related features in own plugin. However, the function of Puppetrun can be subsided by Remote Execution plugin. There is two options:
```

Comment:
What version is it going to be removed? 2.1 still has it. The plugin will only be for extracting ENC functionality, so it's not directly related to this change (other than as part of the overall effort of making puppet optional).
Please also link to the discourse thread discussing this change for further information.